src/TechObject/ObjectsTree/UniversalObject/AttachedObjects.cs

```diff
@@ -185,7 +185,7 @@ public override bool SetNewValue(IDictionary<int, List<int>> newDict)
                 }
             }
 
-            foreach (var replacedObj in replacedObjects?.Keys)
+            foreach (var replacedObj in replacedObjects?.Keys ?? Enumerable.Empty<TechObject>())
```

Comment:
Предлагаю так (меньше ?):
```suggestion
            if (replacedObjects != null)
            {
                foreach (var replacedObj in replacedObjects.Keys)
                {
                    var replacedObjNum = TechObjectManager.GetInstance()
                        .GetTechObjectN(replacedObj);
                    if (oldNumbers.Contains(replacedObjNum))
                    {
                        oldNumbers.Remove(replacedObjNum);
                    }
                }
            }
```